web/netlify.toml

```
@@ -10,7 +10,17 @@ command = "yarn run build"
   VUE_APP_SENTRY_DSN = "https://425b9a012300493d867e97785fae7b88@o308436.ingest.sentry.io/5196549"
   VUE_APP_SENTRY_ENVIRONMENT = "staging"
 
+
+[[redirects]]
+from = "/server-info"
+to = "/server-info.json"
```

```suggestion
to = "https://api.dandiarchive.org/server-info.json"
```
Rather than bake the `server-info.json` file into netlify, why not just use the `status = 200` to proxy for the API server directly?

---

It was a request from @yarikoptic and @satra to not redirect, but to directly serve this information, to account for programmatic access and redirects not necessarily being followed.

---

Oh, I think I see what you mean, e.g. https://docs.netlify.com/routing/redirects/rewrites-proxies/. This would be worth giving a shot.

---

For posterity: This is possible but we can't change this value based on deploy context, so [fd4576d](https://github.com/dandi/dandi-archive/pull/1018/commits/fd4576d8b17e67dd9254f3ae77fb4d9b83c129dc) updates the plugin to proxy the request to the `api/info` endpoint. This should be more robust anyway.